rocky/tools/models.py

```diff
@@ -248,8 +248,11 @@ def has_perm(self, perm: str) -> bool:
     def has_perms(self, perm_list: Iterable[str]) -> bool:
         return all(self.has_perm(perm) for perm in perm_list)
 
-    def has_clearance(self, scan_level: int) -> bool:
-        return self.acknowledged_clearance_level >= scan_level
+    def has_clearance(self, plugin: Plugin) -> bool:
+        perm = self.has_perm("tools.can_enable_disable_boefje")
```

Comment:
should we create a new permission for normalizers too? We are heading towards various plugin types, and should be able to differentiate between them easily.

Reply:
> should we create a new permission for normalizers too? We are heading towards various plugin types, and should be able to differentiate between them easily.

Agree, makes sense

Reply:
I see that we have a `can_scan_organization` which is enough when enabling I think. You enable a plugin so it can start scans automatically. I think we can remove the can enable boefje (refactoring) and start using the general `can_scan_organization`